src/ensembl/src/content/app/species/SpeciesPage.tsx

```diff
@@ -42,9 +40,20 @@ type SpeciesPageParams = {
 
 const SpeciesPage = () => {
   const { genomeId } = useParams() as SpeciesPageParams;
+  const dispatch = useDispatch();
+
+  const changeGenomeId = useCallback(
```

Comment:
- why the useCallback?
- even if we use the useCallback (it's a bit of an optimisation), it wouldn't depend on the genome id.

Reply:
Removed it